README.rdoc

```diff
@@ -46,6 +46,19 @@ circle.yml:
             files:
               - test/**/*.rb
 
+Alternatively, you can add the following code to your test runner setup:
+
+    # test/helper.rb
+    require 'minitest'
+    require 'minitest/autorun'
+    require 'minitest/ci'
+
+    Minitest::Ci.report_dir = "#{ENV["CIRCLE_TEST_REPORTS"]}/reports"
```

Comment:
Should there be a condition around this to keep from writing reports in development? I'm thinking something like:

```ruby
if (report_root = ENV['CIRCLE_TEST_REPORTS'])
  Minitest::Ci.report_dir = "#{report_root}/reports"
end
```